src/linter/report.go

```diff
@@ -263,6 +263,24 @@ func init() {
 			Default: true,
 			Comment: `Report illegal non-public access level in interfaces.`,
 		},
+
+		{
+			Name:    "classConstantRedefinition",
+			Default: true,
+			Comment: `Report redefinition of class constant.`,
+		},
+
+		{
+			Name:    "classPropertyRedeclaration",
+			Default: true,
+			Comment: `Report redeclaration of class property.`,
+		},
+
+		{
+			Name:    "classMethodRedeclaration",
+			Default: true,
+			Comment: `Report redeclaration of class method`,
```

Comment:
I'm not sure that we need 3 different checks for this. They're checking a single logical error.